tests/unit/test_finder.py

```diff
@@ -365,6 +366,27 @@ def test_finder_only_installs_stable_releases(data):
         assert link.url == "https://foo/bar-1.0.tar.gz"
 
 
+def test_finder_only_installs_data_require(data):
+    """
+    """
+
+    # using a local index (that has pre & dev releases)
+    finder = PackageFinder([],
+                           [data.index_url("datarequire")],
```

Comment:
I tried to do that, but it appeared to be extremely painful to figure out how to get the `index.html` to be found by PackageFinder, but I'm not familiar with API.